decidim-initiatives/spec/events/decidim/initiatives/initiative_sent_to_technical_validation_event_spec.rb

```diff
@@ -3,16 +3,18 @@
 require "spec_helper"
 
 describe Decidim::Initiatives::InitiativeSentToTechnicalValidationEvent do
-  let(:resource) { create :initiative }
+  include_context "when a simple event"
+
+  let(:resource) { create :initiative, title: generate_localized_title(:initiative_title) }
```

Comment:
The title parameter wasn't in the original PR. I understand also that's not necessary as we're already defining the default title in the factory
https://github.com/decidim/decidim/blob/e1e6aa682c29a71e4f5f8361e7b57345100da6d7/decidim-initiatives/lib/decidim/initiatives/test/factories.rb#L8

```suggestion
  let(:resource) { create :initiative }
```

Reply:
Actually this is needed. 
![image](https://github.com/decidim/decidim/assets/105683/9e02d035-b914-4dfd-b1ac-183dd3cc103f)